src/librbd/operation/DisableFeaturesRequest.cc

```diff
@@ -636,4 +636,6 @@ Context *DisableFeaturesRequest<I>::handle_finish(int r) {
 } // namespace operation
 } // namespace librbd
 
+#ifndef TEST_F
```

Comment:
This doesn't appear to be necessary -- I successfully built and linked unittest_librbd w/o this change.